windows/src/developer/TIKE/main/UframeTextEditor.pas

```diff
@@ -925,7 +925,9 @@ procedure TframeTextEditor.tmrUpdateSelectedTokenTimer(Sender: TObject);
     if ch = '' then Exit;
 
     if Uni_IsSurrogate2(ch[1]) then
-      ch := memo.GetTextPart(memo.SelStart-2, memo.SelStart);
+      ch := memo.GetTextPart(memo.SelStart-2, memo.SelStart)
```

Comment:
No `;`?

Reply:
That's correct, Pascal does not use a `;` before an `else` - it's considered part of the same `if` statement.